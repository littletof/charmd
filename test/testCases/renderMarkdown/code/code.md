Inline `code` *`italics`* **`bold`** ***`both`***

    // Some comments
    line 1 of code
    line 2 of code
    line 3 of code

```
Sample text here...
```

```ts
function sum(...nums: number[]): number {
  return nums.reduce((sum, num) => sum + num, 0);
}
```