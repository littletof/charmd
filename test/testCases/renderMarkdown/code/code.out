Inline [7m code [27m [3m[7m italics [27m[23m [1m[7m bold [27m[22m [3m[1m[7m both [27m[22m[23m

[47m[30m[3m codeblock [23m[39m[37m         [39m[49m
[100m [49m[100m[90m                   [39m[49m
[100m[90m  [39m[49m[100m[30m[3m// Some comments[23m[39m[49m[100m[90m  [39m[49m
[100m[90m  [39m[49m[100m[30m[3mline 1 of code[23m[39m[49m[100m[90m    [39m[49m
[100m[90m  [39m[49m[100m[30m[3mline 2 of code[23m[39m[49m[100m[90m    [39m[49m
[100m[90m  [39m[49m[100m[30m[3mline 3 of code[23m[39m[49m[100m[90m    [39m[49m
[100m [49m[100m[90m                   [39m[49m

[47m[30m[3m codeblock [23m[39m[37m            [39m[49m
[100m [49m[100m[90m                      [39m[49m
[100m[90m  [39m[49m[100m[30m[3mSample text here...[23m[39m[49m[100m[90m  [39m[49m
[100m [49m[100m[90m                      [39m[49m

[47m[30m[3m codeblock [ts][23m[39m[37m                                      [39m[49m
[100m [49m[100m[90m                                                    [39m[49m
[100m[90m  [39m[49m[100m[30m[3mfunction sum(...nums: number[]): number {[23m[39m[49m[100m[90m          [39m[49m
[100m[90m  [39m[49m[100m[30m[3m  return nums.reduce((sum, num) => sum + num, 0);[23m[39m[49m[100m[90m  [39m[49m
[100m[90m  [39m[49m[100m[30m[3m}[23m[39m[49m[100m[90m                                                  [39m[49m
[100m [49m[100m[90m                                                    [39m[49m
